models/articles.model.js

```diff
@@ -2,7 +2,13 @@ const db = require("../db/connection");
 
 exports.selectArticleId = (article_id) => {
   return db
-    .query("SELECT * FROM articles WHERE article_id = $1;", [article_id])
+    .query(
+      `SELECT articles.*, COUNT(comments.article_id) :: INT AS comment_count
+      FROM articles JOIN comments ON articles.article_id = comments.article_id
```

Comment:
Consider a left join for this to account for when the article has no comments. An inner join will only return rows where a relation exists.

Reply:
Great call, thank you :)
changes now made